Google.Api.Gax/VersionHeaderBuilder.cs

```diff
@@ -62,6 +62,12 @@ public VersionHeaderBuilder AppendAssemblyVersion(string name, System.Type type)
         /// Appends the .NET environment information to the list.
         /// </summary>
         public VersionHeaderBuilder AppendDotNetEnvironment() => AppendVersion("gl-dotnet", s_environmentVersion.Value);
+        
+        /// <summary>
+        /// Whether the name or value that are supposed to be included in a header are valid
+        /// </summary>
+        private static bool IsHeaderNameValueValid(string nameOrValue)
+            => !nameOrValue.Contains(" ") && !nameOrValue.Contains("/");
```

Comment:
Nit: we normally put the `=>` at the end of the previous line.

Reply:
:+1: